docs/tempo/website/grafana-agent/automatic-logging.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Automatic logging
+---
+
+# Automatic logging: Trace discovery through logs
+
+Running distributed tracing systems is very powerful, but it brings its own challenges,
+and one of them is trace discovery.
+Tempo supports finding a trace if you know the trace identifier,
+so we leverage other tools like logs and metrics to discover traces.
+
+Automatic logging provides an easy and fast way of getting trace discovery through logs.
+Automatic logging writes a well formatted log line to a Loki instance for each span, root or process that passes through the tracing pipeline.
```

Comment:
We already have this paragraph on the top-level document and I saw that we've some repetition too on the other deep-dives. 

Thoughts?